vertical-pod-autoscaler/e2e/v1/recommender.go

```diff
@@ -145,6 +147,43 @@ var _ = RecommenderE2eDescribe("Checkpoints", func() {
 	})
 })
 
+var _ = RecommenderE2eDescribe("CronJob Recommendation", func() {
+	f := framework.NewDefaultFramework("vertical-pod-autoscaling")
+
+	ginkgo.It("recommendations for CronJob are served", func() {
```

Comment:
This will render as "CronJob Recommendation recommendations for CronJob are served" in the test description (it glues together the E2E describe and the It). Can you fix it?